src/features/common/components/description-list.tsx

```diff
@@ -15,7 +15,7 @@ export function DescriptionList({ items, dtClassName }: Props) {
     <div className={cn('grid grid-cols-[minmax(min-content,auto)_1fr] gap-x-4 gap-y-1.5')}>
       {items.map((item, index) => (
         <dl key={index} className={cn('grid grid-cols-subgrid col-span-2')}>
-          <dt className={cn(dtClassName)}>{item.dt}</dt>
+          <dt className={cn(dtClassName, 'font-medium')}>{item.dt}</dt>
```

Comment:
```suggestion
          <dt className={cn('font-medium', dtClassName)}>{item.dt}</dt>
```

Ensures the `dtClassName` classes will take precedence.